src/index.js

```diff
@@ -1,8 +1,8 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import { Provider } from 'react-redux';
-import store from './store';
-import App from './pages/Routes';
+import React from "react";
+import ReactDOM from "react-dom";
+import { Provider } from "react-redux";
+import store from "./store";
+import App from "./pages/Routes";
```

Comment:
please remove double quotes, @felixkiryowa

Reply:
I have fixed this